docs/docs_in_html/easyrtc_faq.html

```diff
@@ -95,7 +95,7 @@ <h2>EasyRTC Requirements:</h2>
 
 <h2>Is EasyRTC production ready?</h2>
 
-<p>We have used EasyRTC in production since May of 2013 (see <a href="https://tawk.com" title="tawk.com">https://tawk.com</a>). The included demos include notes about what browsers the various features work in, or how experimental the feature is.</p>
```

Comment:
This was presumably owned by Priologic, the link appears to be broken

Reply:
LGTM